### DISCUSSION TITLE
PR merge 없이 local에서 확인하기

### DISCUSSION THREAD

**Discussion by sujin9** [euphony-io/check-in-listener#34] [General]
https://www.lesstif.com/gitbook/github-pr-pull-request-merge-local-30704954.html

PR 올린 것을 merge하기 전에 직접 실행해보고 싶을 때
해당 PR로 브랜치처럼 checkout하는 방법도 있더라구요!
혹시 도움이 될까 싶어서 공유해봅니다.
하단에 upstream PR 받기 부분 3줄만 보셔도 돼요!ㅋㅋ

**Comment by eunzinn** [comment]
이런 방법도 있었군요!! 깃헙 시작한지 얼마 안돼서 저도 몰랐었는데 감사합니다! 👍

**Comment by Jsueeee** [comment]
짱짱입니다!! 완전 좋아요!

**Comment by YoungSeokHong** [comment]
와 이런 방법이 있었네요 ㅋㅋㅋㅋㅋㅋ 그동안 PR올라오면 언제 합쳐지지하고 맨날 기다렸는데 ㅋㅋㅋㅋㅋ ㅠㅠㅠ
감사합니다!😄

**Comment by dlwlstks96** [comment]
```
git clone -b {branch_name} --single-branch {저장소 URL}
ex) git clone -b javajigi --single-branch https://github.com/javajigi/java-racingcar
```
저는 PR 합쳐지기 전에 먼저 테스트하고 싶을 경우엔 위와 같이 특정 브랜치 clone해왔는데 이런 방법도 있군요!!
새로운 곳에 매번 clone 해오는 방법보다 수진님께서 알려주신 방법이 더욱 효율적일 것 같네요:) 좋은 정보 감사합니다!

**Comment by kuro11pow2** [comment]
꿀팁 감사합니다!
